src/Features/Core/Portable/ExtractMethod/MethodExtractor.Analyzer.cs

```diff
@@ -670,7 +663,7 @@ private bool SelectionContainsOnlyIdentifierWithSameType(ITypeSymbol type)
                     return false;
                 }
 
-                return type.Equals(SelectionResult.GetContainingScopeType());
+                return type.Equals(SelectionResult.GetReturnType(this.CancellationToken));
```

Comment:
`GetContainingScopeType` was a terribly confusing name.  it's now just called GetReturnType, which defers to GetReturnTypeInfo